### PR TITLE
update containerd/kubenet_template file with portmap plugin chain

### PR DESCRIPTION
**Pull request by eriksywu**

### CODE REVIEW

**Comment by eriksywu** [comment]
addressed Azure/AKS#2070

File: parts/linux/cloud-init/nodecustomdata.yml [hunk 1]
```diff
@@ -292,6 +292,11 @@ write_files:
                 "subnet": "{{`{{.PodCIDR}}`}}",
                 "routes": [{ "dst": "0.0.0.0/0" }]
             }
+          },
+          {
+            "type": "portmap",
+            "capabilities": {"portMappings": true},
+            "snat": false
```
Comment (bowang-666) [inline parts/linux/cloud-init/nodecustomdata.yml:299]: any particular reason we need to disable snat?

Comment (eriksywu) [reply parts/linux/cloud-init/nodecustomdata.yml]: Changed to use existing KUBE-MARK-MASQ bitmask for SNAT.

Essentially this rule will have traffic from localhost(127.0.0.1) and the podIP(i.e 10.244.x.x) of be SNAT'ed back to the node IP. Docker clusters have a similar set of rules.

**Review by bowang-666 — APPROVED** [review]

**Comment by xuto2** [comment]
@eriksywu I suggest we put a bit contexts for PRs like this for faster review and for future reference. we may need to config our repo to enforce some basic sanity rule for PR merging to remind us.

**Pull request merged**